### PR TITLE
Example of Energy Dependent Spatial Model

### PR DESCRIPTION
**Pull request by AtreyeeS**
**Description**

This pull request adds an example and a unit test of an energy dependent morphological model.

**Dear reviewer**

### CODE REVIEW

**Comment by AtreyeeS** [comment]
I do not understand the Travis fail here - the tests pass locally on my machine, and I did not change any code base to trigger a change in `test_sherpa`

File: docs/tutorials/models.ipynb [hunk 1]
```diff
@@ -51,7 +51,7 @@
    "outputs": [],
```
Comment (adonath) [inline docs/tutorials/models.ipynb:51]: <p>I think in <code>.evaluation_radius()</code>  we should choose the larger one like  <code>np.max(sigma_1TeV, sigma_10TeV)</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3083/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='89'/>

Comment (adonath) [inline docs/tutorials/models.ipynb:51]: <p>We already have <code>SpatialModel.plot_interactive()</code>,  have tried whether this works. And maybe you could just move the code to <code>SpatialModel.plot_grid()</code>.  I think it's convenient if models can plot themselves...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3083/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='93'/>

Comment (AtreyeeS) [reply docs/tutorials/models.ipynb:51]: Done!

Comment (AtreyeeS) [reply docs/tutorials/models.ipynb:51]: Ok! Done

**Review by adonath — APPROVED** [review]
Thanks a lot @AtreyeeS, test fails are unrelated...I'll go ahead and merge. If there are any issues left we can address them in follow-up PRs.

**Pull request merged**